.changeset/big-pots-destroy.md

```diff
@@ -0,0 +1,5 @@
+---
+"@astrojs/starlight": patch
+---
+
+i18n(zh-cn): Improve the toc title translation
```

Comment:
Thank you!

Suggesting a small change to match the style Sarah has recommended to us for changelogs:

```suggestion
Improves the table of contents title translation in Simplified Chinese
```

(Basic rules are: start with a third-person verb like “improves”, “fixes”, “adds” etc. and use a full sentence. More details: https://contribute.docs.astro.build/guides/new-feature-docs/#changesets)